pkg/config/config_template.yaml

```diff
@@ -690,10 +690,20 @@ api_key:
 ## When running in a memory cgroup, openjdk 8u131 and higher can automatically adjust
 ## its heap memory usage in accordance to the cgroup/container's memory limit.
 ## The Agent set a Xmx of 200MB if none is configured.
-## Note: older openjdk versions and other jvms might fail to start if this option is set
+## Note: older openjdk versions and openjdk 10 and higher as well as other
+## JVMs might fail to start if this option is set.
```

Comment:
> older openjdk versions and openjdk 10 and higher as well as other

so... all openjdk versions? 🤔

Reply:
No actually it was refering to the lines above where `openjdk 8u131` is mentionned but I guess that quite confusing, I'll reword it.

Reply:
yeah it is, i think you can just write this constraint directly (`OpenJDK version < 8u131 or >= 10`)